docs/TablePartitionsStorageGroups.md

```diff
@@ -0,0 +1,421 @@
+After the new storage group transitions to _Available_ state, it can be used to store table partitions.
+For more information, see [Example Deployment](#example) and [ALTER TABLE][6].
+
+### Removing Storage Group
+
+A storage group goes permanently offline for the following reasons:
+
+- The storage group is removed from all database archives with no intent of adding it back to any archive.
+- All archives serving the storage group are damaged or lost.
+
+A storage group that has gone permanently offline must be deleted.
+Deleting a storage group informs the database that it no longer needs to wait for the storage group to come back online.
+
+>**Note**
+> A storage group can only be deleted from the database if it was previously removed from all archives or if the archives serving the storage group are deleted.
+
+Remove the Helm release corresponding to this storage group.
+
+```sh
+helm delete sgeast
+```
+
+Delete all _archive_ Persistent Volume Claims (PVCs) for this storage group.
+
+```sh
+for pvc in $(kubectl get pvc -l domain=nuodb,database=demo,release=sgeast -oname | grep '/archive-volume-'); do
+    kubectl delete $pvc
+done
+```
+
+If [Kubernetes Aware Admin][1] (KAA) is configured, it will automatically delete all archive objects from the domain state that corresponds to the deleted PVCs.
+
+Delete a permanently offline storage group from the database.
+
+>**Warning**
+> Deleting a storage group will remove all table partitions and data stored in the storage group.
+
+```sh
+kubectl exec admin-nuodb-cluster0-0 -- \
+    nuocmd delete storage-group --db-name demo --sg-name sgeast
+```
+
+### Storage Group Maintenance
+
+A storage group goes temporarily offline for the following reasons:
+
+- There is no `RUNNING` SM servicing this storage group.
+- None of the SMs servicing the storage group can become `RUNNING` and maintenance operations must be performed such as `nuoarchive check`.
+
+A storage group that is temporarily offline must be brought into maintenance.
+This will inform TEs and scheduled backups that they no longer need to wait for the storage group to come online.
+
+>**Important**
+> Bringing a storage group into maintenance will exclude it from database backups. Restoring from such a backup could cause data loss or inconsistent data and must be used with caution.
+
+To put a storage group into maintenance, scale down all corresponding SMs and remove all the archives that service it.
+
+```sh
+kubectl scale statefulset -l domain=nuodb,database=demo,release=sgeast --replicas 0
+```
+
+Get the archive IDs that are associated with the storage group.
+The `archive_states` property has all archive IDs associated to this storage group.
+
+```sh
+kubectl exec admin-nuodb-cluster0-0 -- \
+    nuocmd get storage-groups --db-name demo
+```
+
+If [Kubernetes Aware Admin][1] is configured, there will be only one archive associated with the storage group as KAA will automatically remove archives for scaled-down SMs.
```

Comment:
I tend to use the terms "deprovision" or "disable" when talking about archive objects being removed from the domain state _without_ `purge=true`. Should we make it clear that the archive is not actually being removed, and also that no data associated with it has been deleted?

Reply:
Good point.